src/components/toast/toast.component.tsx

```diff
@@ -119,12 +145,11 @@ const Toast = React.forwardRef(
     function renderToastContent() {
       if (!open) return null;
 
-      const toastProps = {
-        isCenter,
-        variant: variant || "success",
-        id,
-        maxWidth,
-      };
+      let toastVariant;
+
+      if (variant !== "notice") {
```

Comment:
Extremely minor but should we not just use the isNotice variable here?
```suggestion
      if (!isNotice) {
```

Reply:
I've actually changed it on purpose because otherwise TS compiler does not remove the `notice` value from the possible `variant` values which ultimately causes a TS error on `ToastStyle` component (as it does not accept `notice` as `variant` value)

Reply:
Okay, a bit frustrating, but fair enough then.